modules/react/combobox/lib/hooks/useComboboxKeyboardTypeAhead.ts

```diff
@@ -22,17 +22,17 @@ import {useComboboxModel} from './useComboboxModel';
  * ```
  */
 export const useComboboxKeyboardTypeAhead = createElemPropsHook(useComboboxModel)(model => {
-  const keySofar = React.useRef('');
+  const keySoFar = React.useRef('');
```

Comment:
I think the other spelling made more sense 😉

Reply:
SOFAR:

> _noun_
> 
> a system in which the sound waves from an underwater explosion are detected and located by three or more listening stations, useful in determining the position at sea of [survivors](https://www.google.com/search?sca_esv=e8b89075e9150def&sxsrf=ADLYWILxuesJAN14vw7rMUFV6xeoRVa9wg:1728871443452&q=survivors&si=ACC90nxMSPeZfdJJjQgDsdZJuFuJHqGJ1Cp6iSAb1ykqwEzDHU8Qu46wubqw7zqubJWF3Ccudp4QaDN3txWbuAboM4cn_KSKdY421fnb6YfouMdx_DyRplk%3D&expnd=1&sa=X&ved=2ahUKEwjr6f6B5IyJAxU3IDQIHUXMBqsQyecJegQIQhAN) of a disaster.